modules/govuk/manifests/app.pp

```diff
@@ -414,8 +414,17 @@
     local_tcpconns_established_critical => $local_tcpconns_established_critical,
   }
 
+  if $ensure == 'absent' {
+    $ensure_service_real = 'absent'
+  } else {
+    $ensure_service_real = $ensure_service ? {
+      undef    => running,
```

Comment:
?

Reply:
If a value for `ensure_service` hasn't been specified, then it's value will be `undef`. This is in the branch of the `if` where the app isn't absent, so set the service to be `running`.

Reply:
So this line is replicating the behaviour that you'd get previously though having `$ensure_service` default to `running`.

Reply:
Riiight.